src/probnum/filtsmooth/gaussfiltsmooth/kalman_utils.py

```diff
@@ -34,6 +36,29 @@ def predict_via_transition(
     )
 
 
+def predict_sqrt(
+    dynamics_model,
+    rv,
+    start,
+    stop,
+    _intermediate_step=None,
+    _linearise_at=None,
+    _diffusion=1.0,
```

Comment:
I'm just wondering: Should the function really have these unused arguments? (`_intermediate_step` and `_diffusion`)

Reply:
oops